src/main/java/com/laborete/LaboreteAPI/controller/UsersController.java

```diff
@@ -0,0 +1,29 @@
+package com.laborete.LaboreteAPI.controller;
+
+import com.laborete.LaboreteAPI.entity.Users;
+import com.laborete.LaboreteAPI.services.UsersService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.UUID;
+
+@RestController()
+@RequestMapping("v1/users")
+public class UsersController {
+    private final UsersService usersService;
+
+    public UsersController(UsersService usersService) {
+        this.usersService = usersService;
+    }
+
+    @RequestMapping(value = "/{uuid}", method = RequestMethod.GET)
+    private ResponseEntity<Users> getUserById(@PathVariable UUID uuid) {
```

Comment:
Suggestion: usually we name variable/parameters/fields of type `UUID` just `id` - but it's up to you.